.github/workflows/prepare-release.yml

```diff
@@ -13,3 +13,34 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: false
+
+      - name: Update Changelog
+        id: update-changelog
+        uses: realm/ci-actions/update-changelog@dac063f8ebd86c201590d07536d6f56f86f66ba9
+        with:
+          changelog: ${{ github.workspace }}/CHANGELOG.md
+
+      - name: Update pubspec.yaml
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: 'version: .*'
+          replace: 'version: ${{ steps.update-changelog.outputs.new-version }}+alpha'
+          include: '**/pubspec.yaml'
+
+      - name: Update realm.podspec
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: "  s.version(\\s+)= '[^']*'"
+          replace: "  s.version$1= '${{ steps.update-changelog.outputs.new-version }}+alpha'"
+          include: '**/realm.podspec'
+
+      - name: Create Release PR
+        uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad #! 3.10.1
+        with:
+          branch: release/${{ steps.update-changelog.outputs.new-version }}
```

Comment:
Where is this variable coming from `steps.update-changelog.outputs.new-version`?

Reply:
It's this step: https://github.com/realm/realm-dart/blob/984017ca461a692541abd068ad95af82ccdba368/.github/workflows/prepare-release.yml#L17-L21

The action is part of our common CI actions: https://github.com/realm/ci-actions/tree/main/update-changelog. Once it runs, it will set the `new-version` variable to the inferred new version from the changelog.